docs/autogen/dest.py

```diff
@@ -105,6 +114,43 @@ def remove_except(path, keep):
                 shutil.rmtree(item)
 
 
+def gen_versions(path: Path, current: str):
+    """Returns a dict compatible with the Pydata versions-format"""
+
+    def version_key(name):
+        match = re.match(r"v(\d+)\.(\d+)\.(\d+)", name)
+        if match:
+            return (1, -int(match.group(1)), -int(match.group(2)), -int(match.group(3)))
+        return (0, name)
+
+    versions = [
+        {
+            "name": "current",
+            "version": f"{current}",
+            "url": "https://xnvme.org/",
+            "preferred": True,
```

Comment:
I would prefer if the "current" wasn't repeated. If I read this correctly we will have:
* current
* 0.7.4
* 0.7.3
* etc...

In my opinion it would be nicer to have:
* 0.7.4 (current)
* 0.7.3
* etc...

Reply:
Currently, the items in the list are simply branch and tag-names based on the convention that, that is what they were named when they were placed in the ``docs/`` folder.

To do something like you mention here then this needs to be done in a different manner, where for the branch-names, then the script would need to parse version-information from the HTML in ``docs/<branch_name>`` for all generated docs in such folders.

How about we just keep it simple as:

```
branches
tags
```

?